inttest/customdomain/customdomain_test.go

```diff
@@ -45,8 +45,9 @@ func (s *CustomDomainSuite) TestK0sGetsUpWithCustomDomain() {
 
 	s.AssertSomeKubeSystemPods(kc)
 
-	s.T().Log("waiting to see CNI pods ready")
+	s.T().Log("waiting to see CNI and konnectivity-agent pods ready")
 	s.NoError(common.WaitForKubeRouterReady(s.Context(), kc), "CNI did not start")
+	s.NoError(common.WaitForDaemonSet(s.Context(), kc, "konnectivity-agent"), "Konnectivity-agent did not start")
```

Comment:
Not sure this will have the desired effect. konnetivity-agent reports readiness even it is not connected to all API servers. It might be better to either have the "pod logs" check here (that's how it's done in other inttests), or add a retry to the failing assertion on line 61.

Reply:
Oh I didn't know that, but there is only one apiserver in this test. So I think it will work as expected.